src/Renci.SshNet/SshCommand.cs

```diff
@@ -503,6 +506,7 @@ private void UnsubscribeFromEventsAndDisposeChannel()
             _channel.ExtendedDataReceived -= Channel_ExtendedDataReceived;
             _channel.RequestReceived -= Channel_RequestReceived;
             _channel.Closed -= Channel_Closed;
+            _channel.Close();

```

Comment:
The dispose of the channel actually closes it as well, so I don't think this is necessary.